lib/i18n/american-english.js

```diff
@@ -377,6 +377,16 @@ function isGoodSentence(sentence) {
     return true;
 }
 
+function isZipcode(word) {
```

Comment:
Are you going to care about British postcodes? The format is 6 characters: 3 numbers and 3 letters, alternated, often displayed as 2 words of 3 characters

Reply:
probably in british-english.js?

Reply:
I guess? I'm still passing en-US as the locale for MultiWOZ, I suppose I should pass en-GB...

Reply:
I'm also not sure we'll copy over all code from american-english into british-english, and then canadian-english, australian-english, south-african-english, nigerian-english... I think we'll have to revisit the language pack concept to pass the locale as a parameter, for the truly country specific stuff (phone number intl prefixes are another case like this)